playbooks/sentry.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Install/update Sentry Crash Reporting
+  hosts: sentry
+  gather_facts: false
+  vars:
+    ansible_user: admin
+    admin_username: admin
+    sentry_port: 9000
+  pre_tasks:
+    # SSH Host key checking
+    # https://stackoverflow.com/a/54735937/982195
+    - name: "Check known_hosts for {{ inventory_hostname }}"
+      delegate_to: localhost
+      ansible.builtin.command: shell ssh-keygen -F {{ inventory_hostname }}
+      register: has_entry_in_known_hosts_file
+      failed_when: false
+      changed_when: false
+      ignore_errors: true
+    - name: Ignore host key for on first run for {{ inventory_hostname }}
+      when: has_entry_in_known_hosts_file.rc == 1
+      ansible.builtin.set_fact:
+        ansible_ssh_common_args: '-o StrictHostKeyChecking=no'
+    - name: Gather Facts
+      ansible.builtin.setup:
+  roles:
+    - sentry
```

Comment:
I moved the preliminary roles (docker,certbot,nginx) to the sentry role's dependencies (in `roles/sentry/meta/main.yml`).  Since the role tasks assume these are installed, it is good to explicitly set them as dependencies, so that the role can be understood and run independently of any playbook.